src/components/Logout/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { logout } from '@opensrp/server-logout';
+import React from 'react';
+import { useHistory } from 'react-router';
+import {
+  DOMAIN_NAME,
+  KEYCLOAK_LOGOUT_URL,
+  OPENSRP_LOGOUT_URL,
+  EXPRESS_OAUTH_LOGOUT_URL,
+  BACKEND_ACTIVE,
+} from '../../configs/env';
+import { getPayloadOptions } from '../../services';
+import Ripple from '../page/Loading';
+
+/** HOC function that calls function that logs out the user from both opensrp
+ * and keycloak
+ */
+export const CustomLogout = (): JSX.Element => {
```

Comment:
I think the more generally accepted way to write this type definition is something like this:

```ts
const Component: React.FC<Props> = (props) => (
```
or where there are no props

```ts
const Component: React.FC = () => (
```
```